docs/agents/hi6200.rst

```diff
@@ -0,0 +1,80 @@
+.. highlight:: rst
+
+.. _Hi6200:
+
+==============
+Hi6200 Agent
+==============
+
+This agent uses Modbus TCP to communicate with the Hi6200 Weight Sensor.
+This agent uses ModbusClient from pyModbusTCP to facilitate the communication.
+The agent is able to communicate over ethernet to read and monitor the net and
+gross weights of the scale.
+
+.. argparse::
+    :filename: ../socs/agents/Hi6200/agent.py
```

Comment:
The path should have a lowercase "H" in "hi6200". This is what's causing the docs build failure.